### DISCUSSION TITLE
SwiperElement: Virtual attribute not working properly in Angular 14

### DISCUSSION THREAD

**Discussion by AlexGarrixen** [nolimits4web/swiper#6401] [Q&A]
Trying to use the virtual attribute on the swiper-container doesn't allow drag the slides.

```html
<swiper-container virtual="true">
  <swiper-slide>Slide 1</swiper-slide>
  <swiper-slide>Slide 2</swiper-slide>
  <swiper-slide>Slide 3</swiper-slide>
  ...
</swiper-container>
```

Playback link: https://codesandbox.io/s/angular-swiper-virtual-5l8zpp?file=%2Fsrc%2Fapp%2Fapp.component.html&fbclid=IwAR0ZKVQOpdPSuZVHEajVMIc8nHUZg6zVD8p6aDKoltY03i9i_di_pxBeaCc

Replicate this in React and it works fine there.

Any clue why this happens in angular ?

**Comment by yigsvnsla** [comment]
#6384
